links/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM gradle:4.4-jdk8-alpine as GRADLE
```

Comment:
We use gradle wrapper for management version of gradle.
But your approach used  version gradle from dockerfile
For example https://github.com/JujaLabs/users/blob/master/Dockerfile
I use simple docker java8 image for build without gradle version

Reply:
will not work, as now we have all services in one repo. Means we need to have Dockerfile inside every folder. At the same time, we dont want to duplicate gradlew etc in every subfolder.

Reply:
yes, you're right, thanks